repose-aggregator/components/filters/herp-filter/src/main/scala/org/openrepose/filters/herp/HerpFilter.scala

```diff
@@ -121,8 +123,17 @@ class HerpFilter @Inject()(configurationService: ConfigurationService,
       else stripHeaderParams(allProjectIds.maxBy(getQuality))
     }
 
-    val projectIds = httpServletRequest.getHeaders(OpenStackServiceHeader.TENANT_ID.toString).asScala
-      .++(httpServletRequest.getHeaders(X_PROJECT_ID).asScala).toTraversable
+    def getSplitHeaders(headerInteractor: HeaderInteractor, headers: List[String]): Traversable[String] = {
+      headers.foldLeft(Traversable[String]()) { (accumulator, current) =>
+        accumulator.++(headerInteractor.getSplittableHeaders(current).asScala)
```

Comment:
Hm, I wonder if you still need to invoke the `++` function using dot notation or if you can just use it in infix notation...